mp/src/game/server/momentum/mom_triggers.h

```diff
@@ -320,6 +322,9 @@ class CTriggerTimerStart : public CTriggerZone
     // front: ref bhop_w1s1
     bool m_bTimerStartOnJump;
     int m_iLimitSpeedType;
+
+    string_t m_strTeleDestName;
+    CBaseEntity *m_pTeleDest;
```

Comment:
EHANDLE would be better, but it probably doesn't matter. Nobody is going to tamper with entity after it's found.